src/python/pants/bin/pants_runner.py

```diff
@@ -75,6 +75,7 @@ def run(self, start_time: float) -> ExitCode:
         # We enable logging here, and everything before it will be routed through regular
         # Python logging.
         setup_logging(global_bootstrap_options, stderr_logging=True)
+        ExceptionSink.set_logging_initialized()
```

Comment:
We call `setup_logging` in a few different places (`pants_runner.py` `daemon_pants_runner.py` and `pants_daemon.py`) because we need to do slightly different things pertaining to logging setup depending on whether pantsd is or is not running. I'm concerned that if we call `set_logging_initialized` here, we won't handle the pantsd case properly.